packages/examples/src/Feedback/FeedbackContainer.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { composeStories } from "@storybook/react";
+import * as stories from "./FeedbackContainer.stories";
+import { render, screen } from "@testing-library/react";
+import * as React from "react";
+import "@testing-library/jest-dom";
+
+const { Primary, Liked, Like, LikeFailure, Loading } = composeStories(stories);
+
+describe("FeedbackContainer", () => {
+  it("should show like button", async () => {
+    render(<Primary />);
+    const button = await screen.findByRole("button", { name: "Like" });
+    expect(button).toBeInTheDocument();
+  });
+
+  it("should show unlike button", async () => {
+    render(<Liked />);
+    const button = await screen.findByRole("button", { name: "Unlike" });
+    expect(button).toBeInTheDocument();
+  });
+
+  it("should show unlike button after clicking like button", async () => {
+    const { container } = render(<Like />);
+    await Like.play({ canvasElement: container });
+    const button = await screen.findByRole("button", { name: "Unlike" });
+    expect(button).toBeInTheDocument();
+  });
+
+  it("should show an error if the like button fails", async () => {
+    const { container } = render(<LikeFailure />);
+    await LikeFailure.play({ canvasElement: container, id: LikeFailure.id });
+    const error = await screen.findByText("Something went wrong");
+    expect(error).toBeInTheDocument();
+  });
+
+  it("should show loading state", async () => {
+    render(<Loading />);
+    const loading = await screen.findByText("Loading...");
+    expect(loading).toBeInTheDocument();
+  });
+});
```

Comment:
Passes, but gives an "act" error for me.

<img width="1536" alt="Screenshot 2023-05-13 at 01 22 44" src="https://github.com/microsoft/nova-facade/assets/2320/28f5a0e9-5535-4bdf-bbea-27d458d0a19c">

Reply:
Will check, I was running `yarn ci` locally, that's why I haven't noticed it :)

Reply:
Fixed!